components/Filterbar/index.js

```diff
@@ -54,44 +53,50 @@ export default function FilterBar({
     <>
       {filterMode ? (
         <StyledFilter>
-          <FilterLabel>
+          <StyledLabel>
             Difficulty:
-            <StyledSelect
-              value={difficultyFilter}
-              onChange={onDifficultyChange}
-            >
-              <option value="">select...</option>
-              <option value="easy">easy</option>
-              <option value="medium">medium</option>
-              <option value="hard">hard</option>
-            </StyledSelect>
-            <ResetButton onClick={handleDifficultyReset}>
-              <Image
-                src={"/assets/delete_FILL0_wght400_GRAD0_opsz24.svg"}
-                alt="Reset difficulty button"
-                width={15}
-                height={15}
-              />
-            </ResetButton>
-          </FilterLabel>
-          <FilterLabel>
+            <Wrapper>
+              <StyledSelect
+                value={difficultyFilter}
+                onChange={onDifficultyChange}
+              >
+                <option value="">select...</option>
+                <option value="easy">easy</option>
+                <option value="medium">medium</option>
+                <option value="hard">hard</option>
```

Comment:
The Selection option could be created dynamically. You use these two times in this file. 🤓
 **But that's really whining on a high level.**